### PR TITLE
Flink files for streamingFileSink

### PR DESCRIPTION
**Pull request by kailashhd**
Allow the class to be public instead of private.

### CODE REVIEW

File: flink-formats/flink-parquet/pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@ under the License.
 	<parent>
 		<groupId>org.apache.flink</groupId>
 		<artifactId>flink-formats</artifactId>
-		<version>1.8-SNAPSHOT</version>
+		<version>1.8-lyft20190407</version>
```
Comment (yxu-valleytider) [inline flink-formats/flink-parquet/pom.xml:28]: nit: the date may need to be updated

**Review by yxu-valleytider — APPROVED** [review]
LGTM.  Need to ping #compute to fast-forward the process.

File: flink-formats/flink-parquet/pom.xml (hunk 1 above)
Comment (tweise) [reply flink-formats/flink-parquet/pom.xml:28]: Why this change?

**Review by tweise — CHANGES_REQUESTED** [review]
Please first get agreement for the upstream change and prepare a pull request. Then this PR needs to be tagged with the JIRA. We need to continue to follow this process to ensure that we are not introducing incompatible changes into the fork that will later make upgrades difficult. The process as such isn't new, please check with @glaksh100 if you have questions.

**Comment by yxu-valleytider** [comment]
There's a solution now which does not require changing Flink upstream.

**Comment by kailashhd** [comment]
@tweise: This is a WIP branch. Not planning on merging this. You can ignore this 
PR for the time being. Will come back to this along with going on the correct process to upstream this change.

**Pull request closed without merging**